src/TestUtils/GcloudWrapper/CloudFunction.php

```diff
@@ -276,7 +276,16 @@ public function run(array $env = [], string $port = self::DEFAULT_PORT, string $
         $this->localUri = 'localhost:' . $port;
 
         $phpBin = $phpBin ?? (new PhpExecutableFinder())->find();
-        $cmd = $phpBin . ' -S ' . $this->localUri . ' vendor/bin/router.php';
+
+        // Opt to use functions framework router, if available.
+        $ff_router = 'vendor/google/cloud-functions-framework/router.php';
+        $default_router = 'vendor/bin/router.php';
+
+        if (file_exists($ff_router)) {
+            $cmd = $phpBin . ' -S ' . $this->localUri . $ff_router;
+        } else {
+            $cmd = $phpBin . ' -S ' . $this->localUri . $default_router;
+        }
```

Comment:
We should keep it simple and stop using the `vendor/bin/router.php` all together, as it no longer works as expected in any context
```suggestion
        $router = 'vendor/google/cloud-functions-framework/router.php';
        $cmd = sprintf('%s -S %s %s', $phpBin, $this->localUri, $router);
```